app/interface/deps/usecases/course_deps.py

```
@@ -0,0 +1,33 @@
+def get_activity_list_usecase(
+    scraper: LmsScraper = Depends(get_lms_scraper, use_cache=False),
+) -> ActivityListUseCase:
+    return ActivityListUseCase(scraper=scraper)
+
+
+def get_course_list_usecase(
+    request: CourseListRequest,
+    scraper: LmsScraper = Depends(get_lms_scraper, use_cache=False),
+) -> CourseListUseCase:
+    return CourseListUseCase(
+        year=request.year, semester=request.semester, scraper=scraper
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)